src/components/scenes/SweepPrivateKeyCompletionScene.tsx

```diff
@@ -183,7 +143,7 @@ const SweepPrivateKeyCompletionComponent = (props: Props) => {
     )
   }, [done, navigation, styles.bottomButton])
 
-  const keyExtractor = useHandler((item: SweepPrivateKeyItem) => item.key)
+  const keyExtractor = useHandler((spendInfo: EdgeTransaction) => spendInfo.tokenId ?? '')
```

Comment:
Shouldn't this be called `tx`, if it's a transaction?

Reply:
Also, if we have `sweepPrivateKeyList` up above, that suggests we have multiple keys, and if we have multiple keys, then potentially we could have overlapping tokenId's (such as the `null` ones). Would it make sense for the key to be `tx.walletId + tx.tokenId` or something like that?